Update contribution guidelines (#3663)
mapfish/mapfish-print: CONTRIBUTING.md

# Contributing to MapFish Print

> Thanks for your interest in contributing to OpenLayers. Please follow Github code of conduct.

Reviewer: ```suggestion
Thanks for your interest in contributing to MapFish Print. Please follow Github code of conduct.
```

`git apply --patch` and `git rebase` can help you create a clean commit
history.
[Reviewboard.org](https://www.reviewboard.org/docs/codebase/dev/git/clean-commits/)
and [Pro GIT](https://git-scm.com/book/en/Git-Tools-Rewriting-History) have
explain how to use them.

### Use clear commit messages

Commit messages should be short, begin with a verb in the imperative, and
contain no trailing punctuation. We follow
https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
for the formatting of commit messages.

Git commit message should look like:

    Header line: explaining the commit in one line

    Body of commit message is a few lines of text, explaining things
    in more detail, possibly giving some background about the issue
    being fixed, etc etc.

    The body of the commit message can be several paragraphs, and
    please do proper word-wrap and keep columns shorter than about
    74 characters or so. That way "git log" will show things
    nicely even when it's indented.

    Further paragraphs come after blank lines.

Please keep the header line short, no more than 50 characters.

### Be possible to merge automatically

> Occasionally other changes to `main` might mean that your pull request cannot

Reviewer: ```suggestion
Occasionally other changes to `master` might mean that your pull request cannot
```

be merged automatically. In this case you may need to rebase your branch on a

> more recent `main`, resolve any conflicts, and `git push --force` to update

Reviewer: ```suggestion
more recent `master`, resolve any conflicts, and `git push --force` to update
```
